packages/api/README.md

```diff
@@ -124,4 +130,16 @@ A few notes about the response.
 - `path` is not stored in the metadata directory; it's inferred from the SPA's directory in the webroot.
 - The "SPAnonymous" app has `null` values for name and ref because it was not deployed with `/deploy`, but it's included so that `/list` provides a complete report of what paths are being made available.
 
+## Scripts
+
+During development, this repo includes a handy script (`npm run get-pubkey`) for downloading your Keycloak server's public key. It accepts the same `auth:keycloak:url` and `auth:keycloak:realm` options as the API itself. You can run it as follows:
+
+```
+npm run get-pubkey -- --auth:keycloak:url https://auth.spaship.io --auth:keycloak:realm SPAshipUsers
+```
+
+_(Just like the API, it also accepts environment variables or a config file instead of CLI options.)_
+
+After running the script, a `.key` file will be saved in your current directory. You can reference that file in the `auth:keycloak:pubkey_file` configuration option.
```

Comment:
should we add it to `.gitignore` ?

Reply:
Good idea.  I have it in my `.git/info/exclude` but I'll copy that into gitignore.